apps/performance-tests/src/util/Datasets.ts

```diff
@@ -75,6 +91,27 @@ export class Datasets {
     return path.resolve(localPath);
   }
 
+  private static getIModelFactory(key: IModelName, elementCount: number) {
+    switch (key) {
+      case "50k elements":
+        return async (name: string, localPath: string) => this.createFlatIModel(name, localPath, elementCount);
+      case "50k nested elements":
+        return async (name: string, localPath: string) => this.createNestedElementModel(name, localPath, elementCount);
+      case "50k group member elements":
+        return async (name: string, localPath: string) => this.createNestedGroupModel(name, localPath, elementCount);
+      case "1k nested subjects":
+        return async (name: string, localPath: string) => this.createNestedSubjectModel(name, localPath, elementCount);
+      case "50k subcategories":
+        return async (name: string, localPath: string) => this.createNestedCategoriesModel(name, localPath, elementCount);
+      case "50k nested functional 3D elements":
+        return async (name: string, localPath: string) => this.createFunctionalElements3dModel(name, localPath, elementCount);
+      case "10k nested functional 2D elements":
+        return async (name: string, localPath: string) => this.createNestedFunctionalElements2dModel(name, localPath, elementCount);
```

Comment:
There's inconsistency with iModel names and their factory function names. I'd rather drop the "nested" term from both - key and function name.